sdk/inc/azure/iot/az_iot_hub_client.h

```diff
@@ -712,12 +712,15 @@ typedef enum
 {
   AZ_IOT_HUB_CLIENT_PROPERTIES_RESPONSE_TYPE_GET
   = AZ_IOT_HUB_CLIENT_TWIN_RESPONSE_TYPE_GET, /**< A response from a properties "GET" request. */
-  AZ_IOT_HUB_CLIENT_PROPERTIES_RESPONSE_TYPE_DESIRED_PROPERTIES
+  AZ_IOT_HUB_CLIENT_PROPERTIES_RESPONSE_TYPE_WRITEABLE
   = AZ_IOT_HUB_CLIENT_TWIN_RESPONSE_TYPE_DESIRED_PROPERTIES, /**< A "PATCH" response with a payload
```

Comment:
We should maybe remove references to "patch" here?
```suggestion
  = AZ_IOT_HUB_CLIENT_TWIN_RESPONSE_TYPE_DESIRED_PROPERTIES, /**< A response with a payload
```